### PR TITLE
TAKE ME TO THE DOWNLOADS

### PR DESCRIPTION
**Pull request by shanselman**
The Readme doesn't mention the store link until HALFWAY DOWN. The CTA here is to get the terminal.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,3 +1,6 @@
+# TL;DR 
+Just want to download the Windows Terminal? Get the [new Temrinal FREE in the Windows Store](https://www.microsoft.com/en-us/p/windows-terminal-preview/9n0dx20hk701) now! It'll automatically be updated with new features they are released.
```
Comment (zadjii-msft) [inline README.md]:
```suggestion
Just want to download the Windows Terminal? Get the [new Terminal FREE in the Windows Store](https://www.microsoft.com/en-us/p/windows-terminal-preview/9n0dx20hk701) now! It'll automatically be updated with new features they are released.
```

**Comment by zadjii-msft** [comment]
I'd also probably not make the tldr a top-level header, or put it above _literally everything else_. I'd maybe nest it under the "Windows Terminal" header that's on line 24. But hey I'm not a PM, @bitcrazed and @cinnamon-msft are.

**Comment by cinnamon-msft** [comment]
I agree with @zadjii-msft. I would prefer a store link right above the Build Status with just a Microsoft Store Download header. Also, we may want to reference the Releases page in case the Store isn't an ideal option 😊

**Comment by miniksa** [comment]
> I agree with @zadjii-msft Mike Griese FTE. I would prefer a store link right above the Build Status with just a Microsoft Store Download header. Also, we may want to reference the Releases page in case the Store isn't an ideal option 😊

It looks like we have a store badge generator here (https://developer.microsoft.com/en-us/store/badges) that makes things like:


<a href='//www.microsoft.com/store/apps/9N0DX20HK701?cid=storebadge&ocid=badge'>English badge</a>

or maybe

<div id="mspb-c1c3rvglogsd" class="9N0DX20HK701">
  <noscript>
    <link rel="stylesheet" type="text/css" href="https://storebadge.azureedge.net/src/fallback-1.7.0.css">
    <div id="msProductBadge" class="mspb-fallback">
      <div class="mspb-primary-image">
       <a href="https://www.microsoft.com/store/apps/9N0DX20HK701?ocid=periscope" target="_blank">
          
        </a>
      </div>
      <div class="mspb-details">
        <h1>Windows Terminal (Preview)</h1>
        <p class="mspb-publisher">Microsoft Corporation</p>
        <p class="mspb-price">See price in store</p>
        <a class="mspb-link" href="https://www.microsoft.com/store/apps/9N0DX20HK701?ocid=periscope" target="_blank">SEE IN STORE</a>
      </div>
    </div>
  </noscript>
</div>
<script src="https://storebadge.azureedge.net/src/badge-1.8.3.js"></script>
<script>
  mspb('9N0DX20HK701', function(badge) {
    document.getElementById('mspb-c1c3rvglogsd').innerHTML = badge;
  });
</script>

**Comment by zadjii-msft** [comment]
> selects the "JavaScript enabled = No" option
> store still generates a Javascript badge

**Comment by shanselman** [comment]
Updated with all these suggestions.

**Pull request merged**